src/entities/webLog.entity.ts

```
@@ -0,0 +1,40 @@
+
+  @Column()
+  merchantId: string;
+
+  @Column()
+  webhookUrl: string;
+
+  @Column({ type: 'enum', enum: ['success', 'failed'] })
+  status: 'success' | 'failed';
+
+  @Column('json')
+  payload: any; // or WebhookPayload interface
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)